src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Navbar from './components/Navbar';
+
+import MainPage from './components/MainPage';
+import LogIn from './components/LogIn';
+
+import Details from './components/Details';
+import Reservations from './components/Reservations';
+import ReserveForm from './components/ReserveForm';
+import AddCars from './components/AddCars';
+import Remove from './components/RemoveCars';
+import Register from './components/signup';
+import ReserveCar from './components/reserveCar';
+import { fetchAllCars } from './store/mainpage_reducer';
+
+function App() {
```

Comment:
- [ ] In Chrome, the app is not rendering for me:

![image](https://user-images.githubusercontent.com/57421823/200627490-ebe1256a-7f7d-4d70-bea9-d0b7e18dd421.png)

I had to install Firefox to review the whole project, however, Chrome is the most popular browser.  Please, consider having a look into it so everybody is able to run your app smoothly 💪